src/SmokeTestAllRoutes.php

```diff
@@ -57,9 +56,9 @@ public function testRoutesDoNotReturnHttp500(string $routeName, Route $route): v
 
             $response = $client->getResponse();
             static::assertLessThan(
-                500,
+                Response::HTTP_INTERNAL_SERVER_ERROR,
                 $response->getStatusCode(),
-                \sprintf('Route "%s" returned a 500 error with HTTP method "%s".', $routeName, $method),
+                \sprintf('Route "%s" returned a %d error with HTTP method "%s". Response content: %s', $routeName, $response->getStatusCode(), $method, $response->getContent()),
```

Comment:
I'm not sure this would be a nice addition. Sometimes you want to test APIs that might return huge JSON strings, sometimes you want to test routes that return binary files, and there might be cases where these are in the Response content while still having an HTTP 500 status code.

Also, default Symfony HTTP 500 will be the debug error message, containing a huge bunch of HTML, CSS and Javascript, all inlined, so it'll be unreadable anyway.

Debugging this failed case can be done in several other ways that don't pollute the test output:

* Big'ol `dd()` right in the code, ugly but quick
* XDebug with your IDE, allowing you to make sure you can create a breakpoint specifically for this failure. XDebug setup's difficulty depends on your project architecture so I can't say whether it's fast or not (but usually, if you work on linux and without docker, it's super quick to setup)
* `curl`-ing the HTTP route from your terminal and manually check what's going on
* Opening the URL from your web browser and looking at the Symfony HTTP error page to see what's going on